calcom/cal.com
# Team Workflows

## What does this PR do?

This PR introduces separate team and personal workflows.

Fixes #6502

Loom Video: https://www.loom.com/share/a454909cc6324195aa605ce479def965

**Environment**: Staging(main branch) / Production

## How should this be tested?

- Creating workflows for teams and user (check if event types dropdown shows the correct event types)
- Filter workflows 
- Workflows in event type settings: 
   - See if correct workflows are listed 
   - Test if all listed workflows can be turned on and of

Also, test if creating and duplicating event types still works as expected => if the dialogs open and have the correct prefilled data
